recipes/poliastro/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set version = "0.5.0" %}
+
+package:
+  name: poliastro
+  version: {{ version }}
+
+source:
+  fn: v{{ version }}.tar.gz
+  url: https://github.com/poliastro/poliastro/archive/v{{ version }}.tar.gz
+  md5: 51dabbe817d2c91d6550ddff854fa0f6
+
+build:
+  number: 0
```

Comment:
Add the following to `build`.

    script: python setup.py install --single-version-externally-managed --record record.txt

Then drop `build.sh`, but keep `bld.bat`.